Source/ACE/Network/Handlers/CharacterHandler.cs

```diff
@@ -168,6 +171,22 @@ private static void CharacterCreateSetDefaultCharacterOptions(Character characte
             character.SetCharacterOption(CharacterOption.ListenToLFGChat, true);
         }
 
+        private static void CharacterCreateSetDefaultCharacterPositions(Character character)
+        {
+            var newStartingPosition = CharacterPositionExtensions.StartingPosition;
+            var newInvalidPosition = CharacterPositionExtensions.InvalidPosition;
+            newStartingPosition.character_id = character.Id;
+            newInvalidPosition.character_id = character.Id;
+            character.SetCharacterPositions(CharacterPositionType.PhysicalLocation, CharacterPositionExtensions.StartingPosition);
+            character.SetCharacterPositions(CharacterPositionType.LifestoneUsed, CharacterPositionExtensions.InvalidPosition);

```

Comment:
I don't think these are really necessary.  I'm OK with having "null" for Positions the character has never tied to.